src/pcapi/domain/demarches_simplifiees.py

```diff
@@ -117,7 +124,7 @@ def _has_requested_state(application: dict, states: datetime) -> bool:
 
 
 def _was_last_updated_after(application: dict, last_update: datetime = None) -> bool:
-    if(not last_update):
+    if (not last_update):
```

Comment:
Quitte à modifier cette ligne, autant supprimer les parenthèses :)

```suggestion
    if not last_update:
```